src/components/library-list/button-list.style.jsx

```diff
@@ -51,6 +53,8 @@ export const OutPutDiv = styled.div`
     color: #FFFFFF;
     transition: max-height 0.5s;
     overflow:hidden;
+    margin-left:auto;
+    margin-right:auto;
```

Comment:
Isso aqui ta estranho, preciso que vc coloque alguns prints para eu entender onde vc alterou a magin. O ideal é sempre usar o flex-box:
https://css-tricks.com/snippets/css/a-guide-to-flexbox/

Reply:
isso deixa o texto centralizado no meio da página. É o jeito que vi a maioria dos guias recomendando... acha necessário trocar?

Reply:
com margin-left e right:
![image](https://user-images.githubusercontent.com/38635424/97241205-2583dd00-17cf-11eb-8a10-dcc71d298614.png)
sem eles e usando justifycontent: center:
![image](https://user-images.githubusercontent.com/38635424/97241259-52d08b00-17cf-11eb-8ae8-16092e632901.png)